sdk/core/azure-core/src/http/curl/curl.cpp

```diff
@@ -1288,8 +1310,8 @@ void CurlConnectionPool::CleanUp()
             }
             else
             {
-              // Got a non-expired connection, all connections before this one are not expired.
-              // Break the loop and continue looping the Pool index
+              // Got a non-expired connection, all connections before this one are not
+              // expired. Break the loop and continue looping the Pool index
```

Comment:
Why are we reformatting some comment lines like this? Is this because of clang-format rules? If not, how come CI didn't catch it?